src/components/Toast/Toast.tsx

```diff
@@ -12,7 +12,7 @@ export type Props = {
    */
   className?: string;
   /**
-   * The child node(s) contains the toast message. Note: the toast message is displayed inside a TextPassage, so children can contain raw HTML
+   * The child node(s) contains the toast message. Note: the toast message is displayed inside a Text, so children can contain raw HTML
```

Comment:
This actually isn't true anymore, so we could remove this comment or add `Text` around `children` to make it come true.